docs/Contributing/Vulnerability-processing.md

```diff
@@ -91,6 +94,19 @@ refreshed on a daily basis.
 Finally, we look at the software inventory of each host and execute the assertions contained in the
 corresponding OVAL file - any match is reported using the same channels as with Windows/Mac OS vulnerabilities
 
+#### Ubuntu Kernel Vulnerabilities
```

Comment:
Thanks @mostlikelee! 

This is looking good. I think we want this to go in the user facing docs and instead of the contributor docs (see spot below) so it's easy to find. Do you think there's a way to distill this into one or two sentences?

Under this sentence:
<img width="713" alt="Screenshot 2024-07-03 at 5 53 45 PM" src="https://github.com/fleetdm/fleet/assets/47070608/de5e9eea-dfbe-4167-8898-fd7fb3b91c4e">

Reply:
UPDATE: I added a one sentence summary to the user facing docs in my latest commit and left your detailed explanation in the contributor docs.